src/OptimisticTokenVotingPlugin.sol

```diff
@@ -207,10 +230,19 @@ contract OptimisticTokenVotingPlugin is
     /// @inheritdoc IOptimisticTokenVoting
     function effectiveVotingPower(uint256 _timestamp, bool _includeL2VotingPower) public view returns (uint256) {
         uint256 _totalVotingPower = totalVotingPower(_timestamp);
+        uint256 _excludedTotal = 0;
+        // Subtract voting power of excluded addresses
+        for (uint256 i = 0; i < excludedVotingPowerHolders.length; i++) {
+            _excludedTotal += votingToken.getPastVotes(excludedVotingPowerHolders[i], _timestamp);
+        }
+
+        // Conditionally subtract taikoBridge if L2 voting is skipped
         if (!_includeL2VotingPower) {
-            return _totalVotingPower - bridgedVotingPower(_timestamp);
+            _excludedTotal += bridgedVotingPower(_timestamp);
```

Comment:
Why don't we transform `bridgedVotingPower()` into `excludedVotingPower()` and make it contain the voting power of all the addresses of the list?

`bridgedVotingPower` is only used for one thing, which makes less sense if other addresses also exist